config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "homes#index"
+  root "monsters#index"
+  resources :monsters
+
```

Comment:
For the routes we want users to be able to navigate to in the React portion of our application, we can set up the routes to point to the same, one template that has the div app like so:
```ruby
  get "/monsters", to: "homes#index"
  get "/monsters/:id", to: "homes#index"
```

That way we don't need to keep creating erb templates with the same div of app!

Reply:
See lines 3/4